commands/run.js

```diff
@@ -0,0 +1,79 @@
+const cli = require('heroku-cli-util')
+const Sanbashi = require('../lib/sanbashi')
+
+let usage = `
+    ${ cli.color.bold.underline.magenta('Usage:')}
+    ${ cli.color.cmd('heroku container:run web bash')} # Runs bash on the local web docker container
+    ${ cli.color.cmd('heroku container:run worker')}   # Runs the container CMD on the local worker container`
+
+module.exports = function (topic) {
+  return {
+    topic: topic,
+    command: 'run',
+    description: 'builds, then runs the docker image locally',
+    variableArgs: true,
+    help: usage,
+    flags: [
+      {
+        name: 'verbose',
+        char: 'v',
+        hasValue: false
+      },
+      {
+        name: 'arg',
+        hasValue: true,
+        description: 'set build-time variables'
```

Comment:
I think we should decouple these commands (and remove this flag). It's not great that all commands need to know about building images.

Reply:
You mean have a `container:build` command and not run here (nor in push) anymore?

Reply:
Yes. We probably need to keep it part of the push command for compatibility but I think we should separate it for new commands (and eventually for the push command too).